java/messages/src/main/java/com/nedap/retail/messages/epcis/v1_1/EpcisDeleteRequest.java

```diff
@@ -0,0 +1,7 @@
+package com.nedap.retail.messages.epcis.v1_1;
+
+public class EpcisDeleteRequest {
+
+    public String location;
+    public String eventType;
```

Comment:
Why open source delete models? This is for our internal usage so I suggest to place it in `retail-commons`